webapi/src/main/scala/org/knora/webapi/routing/admin/ProjectsRouteADM.scala

```diff
@@ -353,7 +372,35 @@ class ProjectsRouteADM(routeData: KnoraRouteData)
     }
 
   /**
-   * returns all members part of a project identified through shortname
+   * Returns all members part of a project identified through UUID
+   */
+  private def getProjectMembersByUuid(): Route =
+    path(projectsBasePath / "uuid" / Segment / "members") { value =>
+      val iri: String = s"http://rdfh.ch/projects/$value"
```

Comment:
This `String` is repeated across you PR on multiple places. Essentially this is domain logic a we should aim at not repeating it.

IMHO this construct should be a single function maybe use the `ProjectIri.make` factory method instead?

Reply:
This is good point, although this is temporary solution, which would be changed most probably when all IRI types will be changed to contain UUID instead of the shortcode. Currently this is only true for Project IRI.